update_conda_lock/action.yml

```diff
@@ -0,0 +1,129 @@
+name: 'Conda Lock Updating Action'
+description: 'Action Pushes a Branch with Conda Lock Update'
+
+inputs:
+  commit_message:
+    description: 'Message of the bump commit'
+    required: true
```

Comment:
I'm not sure about this being check by the backend. I suggest to try it, and check it explicitly in the script instead.